### PR TITLE
CRI-O: make cgroup naming consistent

### PR DESCRIPTION
**Pull request by saschagrunert**
Make the image configurations always refer to `cgroupv[12]` instead of any other way to write it. This does not change any job config and is only done for future maintainability.

PTAL @kubernetes/sig-node-cri-o-test-maintainers

### CODE REVIEW

**Review by sohankunkerkar — APPROVED** [review]
/lgtm

File: jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml [hunk 1]
```diff
@@ -5,6 +5,6 @@ images:
   ubuntu:
     image_family: fedora-coreos-stable
     project: fedora-coreos-cloud
-    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupsv2_hugepages.ign"
+    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupv1_hugepages.ign"
```
Comment (kannon92) [inline jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml:8]: Is this right?

If so, we should have a groups v2 huge pages job and you should change the file name.

File: jobs/e2e_node/crio/latest/image-config-cgroupv1-resource-managers.yaml [hunk 2]
```diff
@@ -4,4 +4,4 @@ images:
     project: fedora-coreos-cloud
     # Using `n1-standard-4` to enable CPU manager node e2e tests.
     machine: n1-standard-4
-    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupsv2.ign"
+    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupv1.ign"
```
Comment (kannon92) [inline jobs/e2e_node/crio/latest/image-config-cgroupv1-resource-managers.yaml:7]: Same here. Is this v1 or v2?

File: jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml (hunk 1 above)
Comment (saschagrunert) [reply jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml:8]: I assume that's a git thing how to handle the diff. We still have `image-config-cgroupv2-hugepages.yaml` which refers the v2 config.

File: jobs/e2e_node/crio/latest/image-config-cgroupv1-resource-managers.yaml (hunk 2 above)
Comment (saschagrunert) [reply jobs/e2e_node/crio/latest/image-config-cgroupv1-resource-managers.yaml:7]: See https://github.com/kubernetes/test-infra/pull/33374#discussion_r1732796002

**Review by kannon92 — COMMENTED** [review]
/approve
/lgtm

File: jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml (hunk 1 above)
Comment (kannon92) [reply jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml:8]: I see. That is what I get for reviewing on my phone.

**Pull request merged**